monitor.py

```diff
@@ -95,4 +96,4 @@ def alive(self):
     for mem_topic, mem_publisher in zip(mem_topics, mem_publishers):
       mem_publisher.publish(UInt64(getattr(vm, mem_topic)))
 
-    rospy.sleep(POLL_PERIOD)
+    rospy.sleep(poll_period)
```

Comment:
Hadde det vore bedre å laga eit rospy.Rate-objekt i initialiseringa og køyra rate.sleep() i staden?

Reply:
Joda, egentlig. Men her modifiserer jeg bare en fork. Så jeg tenker det beste er å holde seg til så små endringer som mulig for enklere å få inn merge upstream. Dette er også bare debug-verktøy så jeg er ikke så bekymret.